### ISSUE TITLE
Change Warp account email address

### ISSUE THREAD

**Issue by dannyneira** [warpdotdev/Warp#4939]
### Dupe Check

- [X] I have [searched Warp feature requests](https://github.com/warpdotdev/warp/issues?q=is%3Aissue+label%3A%22Feature+requests%22) and there are no duplicates

### Describe the solution you'd like?

There are cases when users want to use Warp with a different email address (change jobs, new email address, etc.)

Currently, there is no direct way to change your email address associated with your Warp account. However, you can [export any Warp Drive](https://docs.warp.dev/features/warp-drive#import-and-export) workflows/notebooks from your older account, create a new account with a new email, and then [import](https://docs.warp.dev/features/warp-drive#import-and-export) them to the new account. Any shared blocks associated with the other email/account will not transfer over to the new account. If you keep the older account active and don't delete it, the shared blocks will still exist, but you won't be able to manage them from the new account.

### Is your feature request related to a problem? Please describe.

There is currently no way to change an email associated with Warp account.

### How important is this feature to you?

3

### Warp Internal (ignore) - linear-label:39cc6478-1249-4ee7-950b-c428edfeecd1

None

**Comment by ran-eh** [comment]
It would be great to allow connecting multiple email addresses to the same account (e.g. github allows this).  This way it will be possible to share settings etc. across, e.g., work and personal.

**Comment by bdav** [comment]
I login with github, but the email address displayed is my old work email that I no longer have access to. No idea if I'm receiving emails there. Would be nice to be able to update the email address without creating a different account.

**Comment by rmaclean** [comment]
Same @bdav

**Comment by 1pronik** [comment]
Changing the name is also would be a great feature because when you log in with GitHub, you can’t do anything with your name.

**Comment by Xtrah** [comment]
> I login with github, but the email address displayed is my old work email that I no longer have access to. No idea if I'm receiving emails there. Would be nice to be able to update the email address without creating a different account.

Same issue!

**Comment by ggfevans** [comment]
I have the same issue described here, and it is preventing me from signing up for a paid version of Warp. My account is currently pointed at an email I don't have access to.

**Comment by jmailaender** [comment]
Same issue 👍🏻

**Comment by gcflymoto** [comment]
Changed Github primary email address, logged out of Warp, logged back in (lost my all my workspace, tabs, history, blocks), still the same ancient email I don't have access to anymore

**Comment by rachelslurs** [comment]
> Changed Github primary email address, logged out of Warp, logged back in (lost my all my workspace, tabs, history, blocks), still the same ancient email I don't have access to anymore

thank you for letting me know and sparing me from trying this 😅

**Comment by dannyneira** [comment]
Hey Folks, Thanks for your patience as we're a small team trying to solve complex issues. There are a whole host of reasons why this is hard, so we recommend making Warp accounts with emails you have access to with a [backup/recovery method in place](https://support.google.com/mail/answer/183723).

When it comes to changing to another email, I've noted a workaround on the [original post at the top](https://github.com/warpdotdev/Warp/issues/4939#issue-2309245516) (e.g. export/import Warp drive). Please continue to 👍 this request to show interest and subscribe for any updates. As we improve account management, etc. within Warp we'll post updates on our [changelog](https://docs.warp.dev/getting-started/changelog).

Finally, please avoid spam comments like +1 / 👍 and thanks for using Warp!

**Comment by kizzlah** [comment]
I didn't become active on GitHub until recently. Was playing around with Warp using my OG Gmail account that I'm in the process of moving away from. And logging in with GitHub is just so much more convenient.

**Comment by codefuturist** [comment]
Dear Warp Team,

As much as I like your terminal and I really do, I find it quit disappointing that such a basic function/feature is not provided. I wait since more than a year to change it, because the effort of creating a new account, exporting, importing and then deleting the old account seems just too much work for something I don't get paid for.
I mean I got lots of experience with IT, and I know lots regular users who would just stop using your product because they don't even know how to do it. So I suggest for your own sake, to just build that basic feature, many people take for granted.
I really do see the potential that more regular users will use your product, because AI lowered the barrier pretty hard, but I can't recommend a product to a regular user if it's missing a basic feature.

Also since a year, I can't delete my old email because of this. 🥲

**Comment by dannytlake** [comment]
I thought signing in with github to create my account would be a good idea. However same issue as other folks in here - it associated my Warp account with an email that i'd rather not have it associated with, and now it looks like the only issue it to delete my account and start over - and NOT use github to sign in.

Warp team: I recommend the disabling Github login until this feature is implemented - it is just going to create more frustrated users who enter the same position we're currently in.

**Comment by soundslikeinfo** [comment]
Yes. I am also facing issues where I regularly change my GitHub account's primary email. I found out about this issue within a day of logging into Warp with my GitHub account.

**Comment by ElijahLynn** [comment]
Interesting. Is this becoming an industry-wide thing? Anthropic doesn't allow you to change your email either...

https://support.claude.com/en/articles/8452276-how-do-i-change-the-email-address-associated-with-my-account

> It's not possible to change the email address associated with your Claude account at this time. When creating an account, please make sure you use an email you'll have long-term access to.

**Comment by 825i** [comment]
>Create a service
>Hundreds of thousands of people start using it
>Millions even
>Don't allow you to change your email address
>Literally don't care
>Give only excuses why or simply ignore people
>Fail to understand that your referral rewards and unlocks are tied to your account
>Still don't care anyway
>Some of your users give you the 🖕🏻 and leave to another service
>You still don't care
